Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -609,9 +613,9 @@ namespace AZ
             {
                 finalizeDrawListsTG.AddTask(
                     finalizeDrawListsTGDesc,
-                    [view]()
+                    [view, &finalizeDrawListsTGEvent]()
```

Comment:
For the future, `auto` can be nice but in cases like this where the type is relevant to understand if value-capture is appropriate in a lambda for example, it would be nice to make the type explicit

Reply:
I'm unclear what change you're suggesting here. Lambda captures don't specify a type from my reading [here](https://en.cppreference.com/w/cpp/language/lambda).

Reply:
As in, you're specifying that `view` should be value captured into the lambda, but because I don't know at a glance what `view`'s type is, I don't know if this is correct or not (at a glance)